src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php

```diff
@@ -6,12 +6,31 @@
 use Symfony\Bundle\FrameworkBundle\Client;
 use Symfony\Component\DomCrawler\Crawler;
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Response;
 
 /**
  * @group installer
  */
 class InstallerControllerTest extends WebTestCase
 {
+    /** @var string */
+    private $kernelDir;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->kernelDir = $this->getProvidedData()[0]->getContainer()->getParameter('kernel.project_dir') . '/app';
+    }
+
+    protected function onNotSuccessfulTest($e): void
```

Comment:
Use a better variable name

Reply:
This is a method from phpunit and part of their interface so I can't change it

Reply:
fixed in 0cf43e9 and 93189de